mozilla/blurts-server
# Port breach data fetching cronjob to TS

# References:

Jira: MNTOR-2142

# How to test

Run `npm run dev:cron:db-pull-breaches`; you should see the same output as `npm run cron:db-pull-breaches` on `main`: breaches and breach icons getting fetched and inserted into your database, and logos getting uploaded to S3.

Likewise for first running `npm run build-cronjobs` and then running `npm run cron:db-pull-breaches`.
